pkg/client/typed/dynamic/client.go

```diff
@@ -177,6 +204,29 @@ func (rc *ResourceClient) Watch(opts v1.ListOptions) (watch.Interface, error) {
 		Watch()
 }
 
+// Get gets the resource with the specified name.
+func (src *SubresourceClient) Get(name string) (*runtime.Unstructured, error) {
```

Comment:
Changed the parameter to `resourceName` and added some clarification in the doc.

> How is this expected to be used?

@bgrant0607 This is how I am using it right now. 

```
c.dc.Resource(resource, namespace).Subresource(scaleRef.Subresource).Get(scaleRef.Name)
```

But with a better understanding of the API machinery now, I am reconsidering this. I will see if I can make this less confusing.